### PR TITLE
feat(blue/green): update template for multi-deploy

### PR DESCRIPTION
**Pull request by danielpeach**
## How has this been tested? How can a reviewer test these changes?

# Have your performed the following tests?

Please confirm you have done the following tests to ensure your pull request is ready to be merged..

- [x] If this change modifies the deployment flow have triggered a deployment using this branch (i.e. `build/dist/darwin_amd64/armory deploy start -f <path to file>`)?
- [x] If this change modifes the login flow have you tried logging in and out with this branch (i.e. `build/dist/darwin_amd64/armory login`) and verified the credentials work?
- [x] Have you verified the specific change made in this PR?

### CODE REVIEW

File: cmd/template/bluegreen.go [hunk 1]
```diff
@@ -122,6 +111,31 @@ strategies:
       shutDownOldVersionAfter:
         - pause:
             untilApproved: true
+
+trafficManagement:
+
+   # Determines the targets that this configuration should be applied to.
+   # If omitted, this configuration will be applied to all targets.
```
Comment (aimeeu) [inline cmd/template/bluegreen.go]:
```suggestion
   # If omitted, this configuration is applied to all targets.
```

File: cmd/template/bluegreen.go [hunk 2]
```diff
@@ -122,6 +111,31 @@ strategies:
...
+ - targets: ["<target>"]
+
+   # The Kubernetes traffic management block defines the Kubernetes Service
+   # resources that should be manipulated as part of the deployment.
+   #
+   # You can specify more than one active / preview pair here; pairs will be
```
Comment (aimeeu) [inline cmd/template/bluegreen.go]:
```suggestion
   # You can specify more than one active / preview pair here; pairs are
```

**Review by aimeeu — CHANGES_REQUESTED** [review]
A few minor nits. Prefer to use present tense instead of "will"

**Review by aimeeu — APPROVED** [review]

File: cmd/template/bluegreen.go [hunk 3]
```diff
@@ -122,6 +111,31 @@ strategies:
...
         - pause:
             untilApproved: true
+
+trafficManagement:
+
+   # Determines the targets that this configuration should be applied to.
+   # If omitted, this configuration will be applied to all targets.
+ - targets: ["<target>"]
+
+   # The Kubernetes traffic management block defines the Kubernetes Service
```
Comment (parth-srivastav) [inline cmd/template/bluegreen.go:104]: We should probably mention that Kubernetes traffic management is required for Blue/Green deployment.

**Pull request merged**